src/Microsoft.DotNet.Build.Tasks.net45/project.json

```diff
@@ -8,7 +8,7 @@
     "NuGet.Versioning": "3.6.0-rc-1987",
     "System.Collections.Immutable": "1.1.37",
     "System.Reflection.Metadata": "1.1.0",
-    "System.Runtime.InteropServices.RuntimeInformation": "4.0.0"
+    "System.Runtime.InteropServices.RuntimeInformation": "4.4.0-beta-24806-04"
```

Comment:
This is what I mean by net45 not having an extended closure for that package:
```
<group targetFramework=".NETFramework4.5" />
<group targetFramework=".NETStandard1.1">
	<dependency id="runtime.native.System" version="4.4.0-beta-24807-03" exclude="Compile" />
	<dependency id="System.Reflection" version="4.4.0-beta-24807-03" exclude="Compile" />
	<dependency id="System.Reflection.Extensions" version="4.4.0-beta-24807-03" exclude="Compile" />
	<dependency id="System.Resources.ResourceManager" version="4.4.0-beta-24807-03" exclude="Compile" />
	<dependency id="System.Runtime" version="4.4.0-beta-24807-03" />
	<dependency id="System.Runtime.InteropServices" version="4.4.0-beta-24807-03" exclude="Compile" />
	<dependency id="System.Threading" version="4.4.0-beta-24807-03" exclude="Compile" />
</group>
```
I suppose all its dependencies are satisfied because there aren't any. :smile: I can see why the dep closure could be a problem if we need to upgrade this package on the .NET Core side too, though.